src/main/java/store/Fruit.java

```diff
@@ -0,0 +1,36 @@
+package store;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.time.LocalDate;
+
+public class Fruit {
```

Comment:
What do you think about using Lombok annotation?

Reply:
if it more good practise I can use it

Reply:
It is just one of practice. Your teammates use Lombok and I think it'll good practice for your to use it.